README.md

```diff
@@ -18,6 +18,10 @@ Roadmap:
 - OpenID Connect implementation
 - Support more DID types
 - Support non self-signed credentials
+- Zero Knowledge Proofs
```

Comment:
i think we should scope this before saying its an upcoming feature, an auth system requires knowledge (such as the did). with zkp we cant really get much useful information

Reply:
the idea would be that the application developer would ask for a credential/presentation that proves something about the user (e.g. age) - this is predicated on:
* the developer being able to specify some presentation request as part of the sign-up/sign-on process
* the user being able to specify which credentials to submit in response

Maybe I got ahead of myself in thinking this would be possible?

Reply:
I was under the impression that we already did support presentations with multiple credentials. Is that an API or SDK limitation?

Reply:
no they support it, i mean in this auth system

Reply:
We'll always find a way! 😄